integr8ly/keycloak-operator
# Intly 1347

## Verification
- oc new-project keycloak
- oc apply -f deploy/rbac.yaml
- oc create -f deploy/example/keycloak_min.json
- oc create -f deploy/example/keycloakRealm_min.json
- make code/run NAMESPACE=keycloak
- wait for provision to finish, confirm version is 7.2
